api/go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	github.com/monochromegane/go-gitignore v0.0.0-20200626010858-205db1a8cc00 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/sergi/go-diff v1.2.0 // indirect
+	github.com/sergi/go-diff v1.3.2-0.20230802210424-5b0b94c5c0d3 // indirect
```

Comment:
This non tag dependency was introduced by the new `github.com/go-git/go-git` pkg.
https://github.com/go-git/go-git/blob/v5.13.2/go.mod#L21

A non-tag dependency was introduced for pkg by fixing the bug.
https://github.com/go-git/go-git/issues/254

I found a new release request to go-diff, but I can't find any response from the owner.
https://github.com/sergi/go-diff/issues/144

Reply:
I just left a thumbs-up in the issue too to show community support.